docs/guides/construct-circuits.ipynb

```diff
@@ -349,6 +403,93 @@
     "qc_a.decompose().draw(\"mpl\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "87f1e053",
+   "metadata": {},
+   "source": [
+    "Qubits can be measured using three methods: [`measure`](/api/qiskit/qiskit.circuit.QuantumCircuit#qiskit.circuit.QuantumCircuit.measure), [`measure_all`](/api/qiskit/qiskit.circuit.QuantumCircuit#measure_all) and [`measure_active`](/api/qiskit/qiskit.circuit.QuantumCircuit#measure_active). To learn how to visualize measured results, see the [Visualize results](/guides/visualize-results) page.\n",
```

Comment:
Can we add a header here and a bit of an intro?  I'm not sure if it should be level 2 or 3.  Maybe something like this (I'm not even close to a quantum physicist, so it should be fixed up to be accurate): 

<span id="measure-qubits"></span>
## Measure qubits

Measurements can be used to get the final results from the quantum register to the classical register.  Note that if you are submitting circuits to a Sampler primitive, measurements are required.  However, circuits submitted to an Estimator primitive cannot contain measurements.

Reply:
Added, thanks!